internal/gittuf-git/pull/pull.go

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package pull
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/gittuf/gittuf/internal/gittuf-git/args"
+)
+
+func GittufPull(gitArgs args.Args) error {
+	// 0. Determine appropriate env vars
+	// 1. Pull non-RSL changes
+	// 2. Pull RSL changes
+	// 3. Pull policy changes
+
+	var env = os.Environ()
+
+	// 0
+	if gitArgs.ChdirIdx > 0 {
+		env = append(env, "GIT_DIR="+gitArgs.GlobalFlags[gitArgs.ChdirIdx])
```

Comment:
This may or may not be the git dir in question. The git dir may be that dir + `.git` etc. I'd also check if every git operation can work with just the git dir change (every gittuf specific git operation can be). Off the top of my head, this may not be true for `git mv` etc, which is probably why both git-dir and -C exist? A chdir may be unavoidable if so.